bucoffea/config/vbfhinv.yaml

```diff
@@ -233,7 +231,7 @@ default:
     regionregex: .*
     apply_hf_cuts: True
     hf_pt_thresh: 80
-    region_without_dijet_cuts: False
+    region_without_dijet_cuts: True
```

Comment:
Do we want to keep this on by default? I think it's fine, I was just thinking it might introduce additional runtime + memory constraints. Especially removing the dijet cuts can increase the number of events passing quite a bit - hence making ParticleNet input dict occupy larger amounts of memory.

Reply:
let's keep this on hold. I think that we should check the DNN distributions without these cuts.